samples/snippets/src/main/java/com/example/bigquery/ExportQueryResultsToS3.java

```diff
@@ -43,7 +43,8 @@ public static void main(String[] args) throws InterruptedException {
     // Export result of query to find states starting with 'W'
     String query =
         String.format(
-            "EXPORT DATA WITH CONNECTION %s AS SELECT * FROM %s.%s.%s WHERE name LIKE 'W%%'",
+            "EXPORT DATA WITH CONNECTION `%s` OPTIONS(uri='%s', format='%s') "
+              + "AS SELECT * FROM %s.%s.%s WHERE name LIKE 'W%%'",
             connectionName, destinationUri, format, projectId, datasetName, externalTableName);
```

Comment:
It looks like you are adding at least two parameters, but not actually adding them.

Reply:
they were missing before!

Reply:
LOL - I should have counted earlier - sorry.

Reply:
No thank you for paying attention! Appreciate it :)